src/database/entities/millions-depositor.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity({ name: 'millions_depositors' })
+export class MillionsDepositorEntity {
+    @PrimaryColumn({ type: 'varchar', length: 128 })
```

Comment:
128 chars length is way too much. Size is guessable, bech32 addresses have a fixed size and so does integers.